backend/src/main/java/com/crm/App.java

```diff
@@ -2,15 +2,13 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.context.ApplicationPidFileWriter;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
 @SpringBootApplication
 @EnableSwagger2
 public class App {
     public static void main(String[] args) {
         SpringApplication springApplication = new SpringApplication(App.class);
-        springApplication.addListeners(new ApplicationPidFileWriter());
```

Comment:
This already has been deleted in another task

Reply:
I started this task two weeks ago before special task for this was added, my fault. I discard this change

Reply:
I guess you can leave like it is.

Reply:
done